README.md

```diff
@@ -1,8 +1,20 @@
 # SimpleHypergraphs.jl
 
-Coverage: [![Coverage Status](https://coveralls.io/repos/github/pszufe/SimpleHypergraphs.jl/badge.svg?branch=master)](https://coveralls.io/github/pszufe/SimpleHypergraphs.jl?branch=master) Linux and macOS: [![Build Status](https://travis-ci.org/pszufe/SimpleHypergraphs.jl.svg?branch=master)](https://travis-ci.org/pszufe/SimpleHypergraphs.jl)
+*A simple hypergraphs package for the Julia programming language.*
 
-A simple hypergraphs package for the Julia programming language.
+| **Documentation** | **Build Status** |
+|---------------|--------------|
+|[![][docs-stable-img]][docs-dev-url] | [![Build Status][travis-img]][travis-url][![Coverage Status][codecov-img]][codecov-url] <br/> Linux and macOS [osx-icon] |
 
-Documentation: [![](https://img.shields.io/badge/docs-latest-blue.svg)](https://pszufe.github.io/SimpleHypergraphs.jl/latest)
+## Documentation
 
+- [**DEV**][docs-dev-url] &mdash; **documentation of the most recently tagged version.**
+
+[docs-stable-img]: https://img.shields.io/badge/docs-latest-blue.svg
```

Comment:
is this a link to stable or latest?

Reply:
Apologize explain better the question, this is the image for latest docs that was originally included in the README I have only restyled

Reply:
You can reference stable (most recently tagged version) or latest (master) version of the documentation.
You have `[docs-stable-img` which suggests stable version but then the badge is `docs-latest-blue.svg` which suggests latest version.

See here https://raw.githubusercontent.com/JuliaData/DataFrames.jl/master/README.md
for an example of the difference.